ip/iputil.go

```diff
@@ -48,6 +95,52 @@ func IsIPv4(ips ...interface{}) bool {
 	return true
 }
 
+// Check if an IP address is part of the list of internal IPs we have declared
+// checks for all ipv4 and ipv6 list
+func IsInternal(str string) bool {
+	_, err := newInternalRangeChecker()
```

Comment:
we can initialize the ranger in `init` and panic in case of error or, as an alternative, we can use `https://pkg.go.dev/sync#Once.Do`